projects/core/src/occ/config-loader/occ-config-loader.service.ts

```diff
@@ -58,7 +59,7 @@ export class OccConfigLoaderService {
   /**
    * Initializes the Spartacus config asynchronously basing on the external config
    */
-  loadConfig(): Promise<I18nConfig | SiteContextConfig> {
+  loadConfig(): Promise<I18nConfig | SiteContextConfig | SiteThemeConfig> {
```

Comment:
have considered to make the team a first class citizen of the `SiteContextConfig`? Similar to language and currency, it could be part of the global site context.

Reply:
@Platonn ^^

Reply:
It's a good idea, as discussed in the morning catchup.